Mimir/src/helpers/onlineLog/Parser.php

```diff
@@ -67,11 +69,35 @@ public function parseToSession(SessionPrimitive $session, string $content, $with
         $reader = new \XMLReader();
         $reader->XML($content);
 
+        $kanNakiCache = null;
+        $this->_ankan_cache = array(
+            0 => array(),
+            1 => array(),
+            2 => array(),
+            3 => array());
+
         while ($reader->read()) {
             if ($reader->nodeType != \XMLReader::ELEMENT) {
                 continue;
             }
 
+            if ($reader->localName === 'N') {
+                $meld = $reader->getAttribute('m');
+                $nakiWho = $reader->getAttribute('who');
+                $kanNakiCache = array('m' => $meld, 'who' => $nakiWho);
+            }
+            else if ($reader->localName !== 'DORA') {
```

Comment:
This looks like code style error, but this one can be fixed with `make autofix`

Reply:
fixed